package-lock.json

```diff
@@ -0,0 +1,4627 @@
+{
```

Comment:
This project currently uses Yarn. You should also use yarn to be consistent and remove the pacakge-lock.json.
@ashneeldas2 any opinions.

Reply:
Agreed, my mistake.